### PR TITLE
[ar] Update Arabic localization of tag strings

### PR DESCRIPTION
**Pull request by adowair**
### Describe your changes

This pull request will update the Arabic localization strings
for all tag and tag-search related terms.

### Related issue number or link (ex: `resolves #issue-number`)

No issue assigned.

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by adowair** [comment]
Screenshot 2023-12-27 at 4 24 52 PM

File: i18n/ar.toml [hunk 1]
```diff
@@ -17,9 +17,23 @@ other = "البحث في الموقع"
...
+[ui_see_all_tags]
+other = "إعرض كل الأوسمة"
+[ui_tag]
+other = "وسم"
+[ui_tags]
+other = "أوسمة"
+[ui_search_by_tags]
+other = "تصفح بالأوسمة"
+[ui_tags_intro]
+other = "صنفت جميع المصطلحات بأوسمة مختلفة لتيسير البحث. طالع المصطلحات المصنفة بوسم بالنقر عليه أسفل."
```
Comment (TarekMSayed) [inline i18n/ar.toml:30]: image

Comment (adowair) [reply i18n/ar.toml]: شكراً أ/طارق، لقد عالجت الخطأ في https://github.com/cncf/glossary/pull/2752/commits/8b60ce454eda3f64107344635dfc6a3f43f95b54

File: i18n/ar.toml [hunk 2]
```diff
@@ -17,9 +17,23 @@ other = "البحث في الموقع"
 [ui_in]
 other = "خلال"
 
-# Used in sentences such as "All Tags"
-[ui_all]
-other = "الكل"
+# Phrases for tags
+[ui_see_all_tags]
+other = "إعرض كل الوسوم"
```
Comment (TarekMSayed) [inline i18n/ar.toml:22]: اعرض

File: i18n/ar.toml [hunk 3]
```diff
@@ -17,9 +17,23 @@ other = "البحث في الموقع"
...
+[ui_tags]
+other = "وسوم"
+[ui_search_by_tags]
+other = "تصفح بالوسوم"
+[ui_tags_intro]
+other = "صنفت جميع المصطلحات بوسومٍٍ مختلفة لتيسير البحث. طالع المصطلحات المصنفة بوسم بالنقر عليه أسفل."
+[ui_or_search_by_tags]
+other = "...أو ابحث عن وسم"
+[ui_select_all]
+other = "إختر الكل"
```
Comment (TarekMSayed) [inline i18n/ar.toml:34]: اختر

Comment (adowair) [reply i18n/ar.toml]: شكراً، تم في https://github.com/cncf/glossary/pull/2752/commits/fef5b4c7255434855ba87a2fbf3a74c5a2b3fb46.

**Review by TarekMSayed — APPROVED** [review]

File: i18n/ar.toml (hunk 2 above)
Comment (adowair) [reply i18n/ar.toml]: شكراً، تم في https://github.com/cncf/glossary/pull/2752/commits/fef5b4c7255434855ba87a2fbf3a74c5a2b3fb46.

**Comment by TarekMSayed** [comment]
@arezk84 LGTM

**Comment by adowair** [comment]
Screenshot 2023-12-27 at 5 42 40 PM

**Comment by adowair** [comment]
هل لديك أية تعليقات أ/@arezk84؟

**Review by arezk84 — APPROVED** [review]
Looks Good 👍

**Pull request merged**